### PR TITLE
[volvooncall] Add missing channels and update documentation (#6880)

### PR DESCRIPTION
**Pull request by soleetions**
Added to the VolvoOnCall binding:

Tyre pressure
Plugin Hybrid (PHEV) / Twin engine data
Preclimatization next to Remote heater
Average speed
Brake fluid level
Also updated the readme file

### CODE REVIEW

File: bundles/org.openhab.binding.volvooncall/README.md [hunk 1]
```diff
@@ -15,19 +15,92 @@ Once a VocApi Bridge has been created with according credential, vehicles connec
...
 Once the bridge created, you will be able to launch discovery of the vehicles attached to it.
 
+## Thing Configuration
+
+The 'VolvoOnCall API' bridge uses the owner's email address and password in order to access the VOC Remote API.
+
+
+| Parameter   | Name             | Description                                                              | Required |
+|-------------|------------------|--------------------------------------------------------------------------|----------|
+| vin         | Vin | Vehicle Identification Number of the car                         | yes      |
```
Comment (clinique) [inline bundles/org.openhab.binding.volvooncall/README.md]: Can you align properly this table please ?

File: bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/handler/VehicleHandler.java [hunk 2]
```diff
@@ -13,9 +13,69 @@
...
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_END_POSITION;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_END_TIME;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_START_ODOMETER;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_START_POSITION;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_START_TIME;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_STOP_ODOMETER;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.UNDEFINED;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.UNLOCK;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.UNLOCK_TIME;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.WASHER_FLUID_LEVEL;
```
Comment (clinique) [inline bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/handler/VehicleHandler.java]: Wasn't ```import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.*;``` more readable ?

Comment (soleetions) [reply bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/handler/VehicleHandler.java]: It's a setting in my IntelliJ based on the preferences from the client project in my day job.

If a * import is the preferred way in OH I can change that setting and do a reformat

Comment (clinique) [reply bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/handler/VehicleHandler.java]: Yes, I do not remember seeing such import examples in OH

**Comment by Jason2866** [comment]
Please merge soon. Thx!

**Comment by soleetions** [comment]
@kaikreuzer, @J-N-K, @Hilbrand, how can I add you as a reviewer for this PR? 
Or is mentioning you in this comment enough?
Thanks!

**Pull request merged**